avae/loss.py

```diff
@@ -97,6 +97,12 @@ def __init__(self, device, beta, gamma=1, lookup_aff=None, recon_fn="MSE"):
         if lookup_aff is not None:
             self.gamma = gamma
             self.affinity_loss = AffinityLoss(lookup_aff, device)
+        elif lookup_aff is None and gamma != 0:
```

Comment:
this means that if we don't provide an affinity.csv file, the code has to be setup to be a standard vae, right? Might be useful to to document that somewhere outside the code.

Reply:
Yep, that's correct. If you set --gamma 0 and -pd 0 that runs just a standard beta-VAE. This used to be quite redundant and  if the matrix was present it used to actually calculate the affinity loss and multiply it with gamma=0 so it wouldn't contribute to the total loss but this change pretty much removes this reduncancy.

I'll add a note of this to the README.md.